x-pack/plugins/enterprise_search/public/applications/shared/kibana_chrome/set_chrome.tsx

```diff
@@ -121,6 +121,23 @@ export const SetAppSearchChrome: React.FC<SetChromeProps> = ({ trail = [] }) =>
   return null;
 };
 
+export const SetEsreChrome: React.FC<SetChromeProps> = ({ trail = [] }) => {
+  const { setBreadcrumbs, setDocTitle } = useValues(KibanaLogic);
+
+  const title = reverseArray(trail);
+  const docTitle = appSearchTitle(title);
+
+  const crumbs = useGenerateBreadcrumbs(trail);
+  const breadcrumbs = useAppSearchBreadcrumbs(crumbs);
```

Comment:
you'll need to write a `useEsreBreadcrumbs` or possibly use `useEnterpriseSearchBreadcrumbs` directly, which is what we did for search applications.

Reply:
Fided in https://github.com/elastic/kibana/pull/159589/commits/0ef388a72252effd27d76180ace3eda5e966d105.